src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -244,5 +245,25 @@ public StudyController(StudyService studyService) {
             return ResponseEntity.ok().headers(header).contentType(MediaType.APPLICATION_OCTET_STREAM).body(exportNetworkInfos.getNetworkData());
         });
     }
+
+    @PostMapping(value = "/{userId}/studies/{studyName}/loadflow/parameters/set")
+    @ApiOperation(value = "set loadflow parameters on study", produces = "application/json")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The loadflow parameters are set")})
+    public ResponseEntity<Mono<Void>> setLoadflowParameters(
+        @PathVariable("studyName") String studyName,
+        @PathVariable("userId") String userId,
+        @RequestBody(required = false) LoadFlowParameters lfParameter) {
+        return ResponseEntity.ok().body(studyService.setLoadFlowParameters(studyName, userId, lfParameter));
+    }
+
+    @GetMapping(value = "/{userId}/studies/{studyName}/loadflow/parameters/get")
```

Comment:
This is not a standard REST API design just a get on /{userId}/studies/{studyName}/loadflow/parameters

Reply:
done